### PR TITLE
Add link to varc and fix Contributing link

### PR DESCRIPTION
**Pull request by chrisdoman**
Hi,

This PR adds a link to a lightweight memory forensics tool (varc) - note I'm one of the authors so have a bit of a bias here :)

Also fixes the link to the CONTRIBUTING page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Introduce commercial and open source tools for memory acquisition.
 - [FEX Memory Imager](https://getdataforensics.com/product/fex-memory-imager/) - FEX Memory Imager (FEX Memory) is a free imaging tool designed to capture the physical Random Access Memory (RAM) of a suspect's running computer. This allows investigators to recover and analyze valuable artifacts found only in memory.
 - [MacQuisition](https://www.blackbagtech.com/category/blog/macquisition/)
 - [Digital Collector](https://cellebrite.com/en/digital-collector/) - A powerful forensic imaging software solution to perform triage, live data acquisition and targeted data collection for Windows and Mac computers.
+- [varc (Volatile Artifact Collector)](https://github.com/cado-security/varc) - varc collects a snapshot of volatile data from a system.
```
Comment (digitalisx) [inline README.md]: Does we need to use a full name? For now, I would like to put the tool name in front of the sentence for clarity.

```suggestion
- [varc](https://github.com/cado-security/varc) - varc collects a snapshot of volatile data from a system.
```

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ A curated list of awesome Memory Forensics for DFIR.
 
 > [Memory Forensics](https://en.wikipedia.org/wiki/Memory_forensics) is forensic analysis of a computer's memory dump. Its primary application is investigation of advanced computer attacks which are stealthy enough to avoid leaving data on the computer's hard drive. Consequently, the memory (RAM) must be analyzed for forensic information.
 
-If you want to contribute, please read the [contribution guidelines](contributing.md).
+If you want to contribute, please read the [contribution guidelines](CONTRIBUTING.md).
```
Comment (digitalisx) [inline README.md:7]: Okay, thank you for correcting my mistake.

**Review by digitalisx — COMMENTED** [review]
First of all, thank you for your contribution. :)

It's not verified how useful this tool is and how it works.. But there seems to be no problem with the registration at this moment. (Because I don't want to give the wrong information to anyone referring to this repo, so I'm careful.)

I left some comment about this PR :)

**Comment by chrisdoman** [comment]
Apologies for the delay and thanks for the edit.
Sure - no problem if you decide not to add it. It's a pretty simple tool that pulls open process memory using the pymem tool, along with dumping a bunch of live system data to a json file. I can create another PR with just the fix to CONTRIBUTING.md instead if you prefer.

Thanks,

**Review by digitalisx — DISMISSED** [review]
All right, open-source contributions always need to be relaxed.
I'll happily merge this!

**Review by digitalisx — APPROVED** [review]

**Pull request merged**